code/obj/item/bruisepack_ointment.dm

```diff
@@ -13,17 +13,8 @@
 	stamina_crit_chance = 3
 
 	examine()
-		set src in view(1)
-		set category = "Local"
-
-		if (src.amount <= 0)
-			qdel(src)
```

Comment:
This doesn't belong in examine code imo, except for novelty items which this surely isn't.